examples/react_example/public/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
If you want you can change the tab icon from the default react one to whatever you want. Just include the picture in your public folder and replace favicon.ico with the picture.